File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.py
def solution(numbers):
    answer=0
    numbers = list(map(str,numbers))
    ## numbers의 범위가 3자리이므로 3자리까지만 비교
    numbers.sort(key = lambda x : x*3, reverse=True)
    
    return str(int(''.join(numbers)))

File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.js
function solution(numbers) {
    var answer = ''
    const strNumbers = numbers.map((number)=>number+'')
    strNumbers.sort((a, b) => (b + a).localeCompare(a + b));
    if (strNumbers[0] === "0") return "0";
    answer = strNumbers.join('')
    return answer;
}